### DISCUSSION TITLE
Refresh token grace period configuration.

### DISCUSSION THREAD

**Discussion by love-bhardwaj** [ory/.github#64] [Q&A] [answered]
Hi, 

I see that the token revocation storage interface has the method [RevokeRefreshTokenMayBeGracePeriod](https://github.com/ory/fosite/blob/45a6785cc54fcbd9195b0de4b821bb5fed6a41be/handler/oauth2/revocation_storage.go#L34). The Go doc for this particular method says `If the Refresh Token grace period is greater than zero in configuration the token` but I haven't been able to find any configuration for the grace period. Could someone please point me to where this configuration lives?

**Comment by vinckr** [comment]
Hey, I am not very familiar with Fosite, but I think this is left to the server, for example in [Ory Hydra](https://www.ory.sh/docs/oauth2-oidc/refresh-token-grant#change-refresh-token-lifespan) you can configure the refresh TTL. 
Also see this discussion, its related maybe: https://github.com/ory/hydra/issues/1831

**Reply by love-bhardwaj** [reply, marked as answer]
From what I have found till now, the grace period is not configurable through the Fosite configuration. The implementor will need to implement it.
